src/facets/NaymsTokenFacet.sol

```diff
@@ -93,16 +46,7 @@ contract NaymsTokenFacet is Modifiers {
     function approve(address spender, uint256 amount) external returns (bool) {
         AppStorage storage s = LibAppStorage.diamondStorage();
```

Comment:
I guess we don't need the appstorage ref here any more. We anyway don't want to access it directly from facets, in general.

Reply:
fixed